Source/CarShack/Startup.cs

```diff
@@ -53,15 +49,19 @@ public void ConfigureServices(IServiceCollection services)
         {
             var builder = services.AddMvcCore(options =>
             {
-                options.OutputFormatters.Clear();
-                options.OutputFormatters.Add(new JsonOutputFormatter(
-                    new JsonSerializerSettings
-                    {
-                        NullValueHandling = NullValueHandling.Ignore,
-                        DefaultValueHandling = DefaultValueHandling.Ignore
-                    }, ArrayPool<char>.Shared));
+                //options.OutputFormatters.Clear();
```

Comment:
This can be removed. Is System.Text doing the same behavior? So no null values and no default values serialized? Or was it intentionally that we wanna send those values? Save bandwidth vs. be explicit.